src/components/FiatOnrampModal/utils.ts

```diff
@@ -0,0 +1,74 @@
+import { ChainId, WETH9 } from '@uniswap/sdk-core'
+import {
+  BRIDGED_USDC_ARBITRUM,
+  MATIC,
+  USDC_ARBITRUM,
+  USDC_MAINNET,
+  USDC_OPTIMISM,
+  USDC_POLYGON,
+  USDT,
+  WBTC,
+  WETH_POLYGON,
+} from 'constants/tokens'
+import { Chain } from 'graphql/data/__generated__/types-and-hooks'
+import { validateUrlChainParam } from 'graphql/data/util'
+
+import { MoonpaySupportedCurrencyCode } from './constants'
+
+type MoonpaySupportedChain = Chain.Ethereum | Chain.Polygon | Chain.Arbitrum | Chain.Optimism
+const moonPaySupportedChains = [Chain.Ethereum, Chain.Polygon, Chain.Arbitrum, Chain.Optimism]
+
+const CURRENCY_CODES: {
```

Comment:
kind of got lost with MoonpaySupportedChain, moonPaySupportedChains, CURRENCY_CODES, MOONPAY_SUPPORTED_CURRENCY_CODES all being separate things. think a comment would help with readability! but otherwise lgtm